website/docs/r/synthetics_monitor.html.markdown

```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
 page_title: "New Relic: newrelic_synthetics_monitor"
 sidebar_current: "docs-newrelic-resource-synthetics-monitor"
 description: |-
-  Create and manage a Synthetics monitor in New Relic.
+Create and manage a Synthetics monitor in New Relic.
 ---
 
 # Resource: newrelic\_synthetics\_monitor
 
-Use this resource to create, update, and delete a synthetics monitor in New Relic.
+Use this resource to create and manage New Relic Synthetics monitor.
 
 ## Example Usage
```

Comment:
Can we also add one example with a private location? Using the private location resource to create one and then link to the monitor.

Reply:
Sure..Will add another example as well